src/tests/pam_test_client.c

```diff
@@ -49,10 +49,10 @@ static struct pam_conv conv = {
     NULL
 };
 #else
-static int dummy_pam_conv(int num_msg,
-                          const struct pam_message **msgm,
-                          struct pam_response **response,
-                          void *appdata_ptr)
+int dummy_pam_conv(int num_msg,
+                   const struct pam_message **msgm,
```

Comment:
Yup, seems to work, thanks!

> On 24 May 2016, at 21:28, Pavel Reichl <notifications@github.com> wrote:
> 
> In src/tests/pam_test_client.c:
> 
> > @@ -49,10 +49,10 @@ static struct pam_conv conv = {
> >      NULL
> >  };
> >  #else
> > -static int dummy_pam_conv(int num_msg,
> > -                          const struct pam_message **msgm,
> > -                          struct pam_response **response,
> > -                          void *appdata_ptr)
> > +int dummy_pam_conv(int num_msg,
> > +                   const struct pam_message **msgm,
> 
> removing const fixed the warning for me...if it was just a single * it would work but there's some magic if it's **...I'm too tired to check now, sorry,,
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>